src/style.css

```diff
@@ -0,0 +1,119 @@
+*,
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/165077714-52111f66-a003-43c4-9f58-fdc884515716.png)

Therefore:

- [ ] Your styling is beautiful, however, please, adjust your CSS to match and clone the to-do list shown in the video.
Most of the work has been already done! 💪

Reply:
Thank you for your reviews but according to this comment I need this project to add to my portfolio so that's why I worked already on the styling 😃